sql/server.go

```diff
@@ -197,6 +197,10 @@ func (s *Server) exec(req driver.Request) (driver.Response, error) {
 			}
 			result.Rows = append(result.Rows, row)
 		}
+		if err := plan.Err(); err != nil {
```

Comment:
Hmm, probably not. I noticed this because subqueries weren't causing errors. Let me add a test case for it.